mne/chpi.py

```diff
@@ -603,284 +559,156 @@ def _fit_cHPI_amplitudes(raw, time_sl, hpi, fit_time, verbose=None):
         # the first component holds the predominant phase direction
         # (so ignore the second, effectively doing s[1] = 0):
         sin_fit[fi, :] = vt[0]
```

Comment:
This seems to return unit strength amplitudes for each coil and each time point. would it be possible to maintain that information. That would let you look at relative hpi signal strength between coils and over time. It would also make the amplitude of the magnetic moment that we end up fitting meaningful. 

Without this `compute_chpi_amplitudes` is much less useful for QA

Reply:
Commit pushed to multiply `s[0]` and ensure that the resulting `slopes` are at least somewhat reasonable